src/app/information/information.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+
+
+
+@Component({
+  selector: 'app-information',
+  templateUrl: './information.component.html',
+  styleUrls: ['./information.component.sass']
+})
+export class InformationComponent implements OnInit {
+
+  name: FormGroup;
+  email: FormGroup;
+  phone: FormGroup;
+  inputValue = '';
+  showImage = false;
```

Comment:
to much variables
use one state variable for state
`state: 'email' | 'name' | 'phone' = 'name';`